Formula/kubernetes-service-catalog.rb

```diff
@@ -0,0 +1,30 @@
+class KubernetesServiceCatalog < Formula
+  desc "Consume services in K8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
+      :revision => "87a5db0e1e0359ce372037a235ce4448944c8611"
+
+  depends_on "coreutils" => :build
```

Comment:
This shouldn't be necessary. Naked calls to `realpath` indicate the software is not properly ported to macOS. Please file an issue upstream.

Reply:
Since we check in our generated files, you don't need to do any make calls at all in order to do a build for homebrew. The `go build` command you have further down should be sufficient.

Let me know if that doesn't work and I'll help get anything fixed on our side (service catalog), so that we can distribute the svcat client with brew! 💯  I'm super excited about this.

Reply:
Awesome @carolynvs. It worked like a charm.